web/src/composebox_typeahead.ts

```diff
@@ -930,7 +926,7 @@ export function get_candidates(
         // Stream regex modified from marked.js
         // Matches '#**stream name** >' at the end of a split.
         const stream_regex = /#\*\*([^*>]+)\*\*\s?>$/;
-        const should_jump_inside_typeahead = stream_regex.test(split[0]);
+        const should_jump_inside_typeahead = stream_regex.test(current_token);
```

Comment:
Why is this change needed? Can you give me an example of it breaking if you don't make this change?